Storage/SQL/FaviconsTable.swift

```diff
@@ -72,7 +72,7 @@ class FaviconsTable<T>: GenericTable<Favicon> {
         if (cursor.count != 1) {
             return nil
         }
-        return (cursor[0] as Favicon).id
+        return (cursor[0] as? Favicon)?.id
```

Comment:
If this isn't a Favicon, something is wrong. I hate this. I'm hoping the Swift fixes will make it better to switch to using more generics. I just gave up originally because they resulted in so many segfaults.